docs/index.rst

```diff
@@ -6,13 +6,13 @@
 
 .. toctree::
    :caption: API Documentation
-   :titlesonly:
    :hidden:
 
-   langs/python
-   langs/arduino
-   langs/cpp
-   langs/matlab
+   api_docs/getting_started_with_apis
```

Comment:
for some reason we set this to titles only which meant no subsections. That was weird, and I needed them, so I've changed it. I also renamed the langs folder to api_docs to be clear.